test/provision/k8s_install.sh

```diff
@@ -38,6 +38,9 @@ fi
 go get github.com/subfuzion/envtpl
 mv ~/go/bin/envtpl /usr/local/bin/
 
+go get github.com/ksonnet/kubecfg
```

Comment:
Please update the contributing guide to include this as a dependency for those who want to run tests in their local environments.

Reply:
done!